lib/api/controllers.js

```diff
@@ -0,0 +1,35 @@
+/*
+ * Kuzzle, a backend software, self-hostable and ready to use
+ * to power modern apps
+ *
+ * Copyright 2015-2020 Kuzzle
+ * mailto: support AT kuzzle.io
+ * website: http://kuzzle.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+module.exports = {
+  Admin: require('./controller/admin'),
+  Auth: require('./controller/auth'),
+  Bulk: require('./controller/bulk'),
+  Collection: require('./controller/collection'),
+  Document: require('./controller/document'),
+  Index: require('./controller/index'),
+  Ms: require('./controller/memoryStorage'),
+  Realtime: require('./controller/realtime'),
+  Security: require('./controller/security'),
+  Server: require('./controller/server'),
```

Comment:
I would rather match the export with the real class name. Also the file should be renamed to match the class declared inside.

Then you will be able to rename the `api/controllers.js` file to `api/controllers/index.js` to follow the Node.js module naming convention.

```suggestion
  AdminController: require('./controllers/adminController'),
  AuthController: require('./controllers/authController'),
  BulkController: require('./controllers/bulkController'),
  CollectionController: require('./controllers/collectionController'),
  DocumentController: require('./controllers/documentController'),
  IndexController: require('./controllers/indexController'),
  MemoryStorageController: require('./controllers/memoryStorageController'),
  RealtimeController: require('./controllers/realtimeController'),
  SecurityController: require('./controllers/securityController'),
  ServerController: require('./controllers/serverController'),
```

Reply:
Ofc, having a **controllers/index.js** is my main intention.

Reply:
Also, there is an inconsistency between our controllers naming and its **sdk.** counterparts.

Like with our  'alias' for MemoryStorage here:
https://github.com/kuzzleio/kuzzle/blob/master/lib/api/funnel.js#L111
And the whole DocumentController -> _document_.

So I thought this turnaround would spare me from doing name parsing in the **proxyNaming** at the **impersonatedSdk level**.